src/downshift.js

```diff
@@ -369,7 +369,6 @@ class Downshift extends Component {
       setHighlightedIndex,
       clearSelection,
       clearItems,
-      itemToString,
```

Comment:
Let's restore this and add the `id` below it.

Then above this line let's leave a blank line and then a comment that says: `// props` like the other sections.